### PR TITLE
Improve software report to print information about Xcode release type

### PR DESCRIPTION
**Pull request by maxim-lobanov**
# Description
Improvement
We should mark beta versions of Xcode in software documentation with `(beta) postfix.

#### Related issue:
https://github.com/actions/virtual-environments-internal/issues/1228

## Check list
- [x] Related issue / work item is attached
- [x] Tests are written (if applicable)
- [x] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

**Review by al-cheb — APPROVED** [review]

File: images/macos/helpers/Xcode.Helpers.psm1 [hunk 1]
```diff
@@ -45,6 +45,14 @@ function Switch-Xcode {
     Invoke-Expression "sudo xcode-select --switch ${XcodeRootPath}"
 }
 
+function Test-XcodeStableRelease {
+    param([string] $XcodeRootPath)
```
Comment (miketimofeev) [inline images/macos/helpers/Xcode.Helpers.psm1:49]: Mandatory?

**Review by dmitry-shibanov — APPROVED** [review]

**Review by vsafonkin — APPROVED** [review]

**Review by dibir-magomedsaygitov — APPROVED** [review]

**Review by Darleev — APPROVED** [review]

**Review by jopdE6vovgedsahzux — APPROVED** [review]

**Review by jopdE6vovgedsahzux — APPROVED** [review]

**Review by AlenaSviridenko — APPROVED** [review]

**Pull request merged**